packages/core/src/components/dialog/_dialog.scss

```diff
@@ -71,7 +71,7 @@ $dialog-padding: $pt-grid-size * 2 !default;
   width: $pt-grid-size * 50;
   padding-bottom: $pt-grid-size * 2;
   pointer-events: all;
-  user-select: text;
+  user-select: initial;
```

Comment:
problem: `initial` is not supported on IE. https://caniuse.com/#feat=css-initial-value
we must have an IE-friendly solution here.

maybe we can use `auto`, the initial value according to MDN (https://developer.mozilla.org/en-US/docs/Web/CSS/user-select)?

Reply:
(this exact issue came up inhttps://github.com/palantir/blueprint/pull/2927)

Reply:
@giladgray how about getting rid of the "user-select" attribute entirely? I could also switch it to auto. I don't really understand why it's there in this case?

Reply:
it's there to reset the overlay's `user-select: none` which is important to prevent interactions with elements behind the backdrop.